egs/reverb/asr1/run.sh

```diff
@@ -68,6 +68,11 @@ minlenratio=0.0
 ctc_weight=0.3
 recog_model=model.acc.best # set a model to be used for decoding: 'model.acc.best' or 'model.loss.best'
 
+# Dereverberation Measures
+# please make sure that you or your institution have the license to report PESQ before turning on the flag
+compute_se=false # flag for turing on computation of dereverberation measures
```

Comment:
Same comment for the Kaldi recipe. Make all metrics except for PESQ default.

Reply:
I am downloading and using the official scripts from REVERB challenge with very little patching. This might require significant patching, if you feel it is very important I can do that.

Reply:
I feel like it is required. Please first try, and if it requires a lot of changes, we could keep the current solution.

Reply:
Sorry for the delay, I didn't notice this comment. I have made this modification now.